mods/ra/rules/misc.yaml

```
@@ -346,7 +346,7 @@ powerproxy.parabombs:
 	AirstrikePower:
 		Icon: parabombs
 		Name: Parabombs (Single Use)
-		Description: A Badger drops a load of parachuted bombs on your target.
+		Description: A Badger drops a load of parachuted bombs\nat a selected location.
```

```suggestion
		Description: A Badger drops a load of parachuted bombs\nat the selected location.
```
to emphasize that it's not any location, but the one you select.

---

Done